mysql-test/suite/sys_vars/r/separate_conn_handling_thread_basic.result

```diff
@@ -1,13 +1,13 @@
 SELECT @@GLOBAL.separate_conn_handling_thread;
 @@GLOBAL.separate_conn_handling_thread
-0
+1
 0 Expected
```

Comment:
Do we need to update this line in the test file?  Should it be "1 expected"?  And at line 11 and 24.